### PR TITLE
Afilias donuts migration

### PR DESCRIPTION
**Pull request by pc-stein**

### CODE REVIEW

File: lib/Net/DRI.pm [hunk 1]
```diff
@@ -284,6 +284,7 @@ sub new
      'Verisign::NAME' => 'VeriSign::NameStore',
      'VNDS' => 'VeriSign::COM_NET', # warning, this was actually split into COM_NET and NameStore!
      'WS' => 'GDI',
+     'Afilias::Afilias' => 'Donuts' # TLds moved to Donuts but extentions only work if using Afilias::Shared
```
Comment (thedarkwinter) [inline lib/Net/DRI.pm:287]: What does this mean actually? Are the Afilias extensions supported in Donuts?

This is also a messy place to put these comments, if any explanation is required it should really be in the DRD file, but I probably wouldn't overcomplicate it and just ignore unless its critical :)

File: lib/Net/DRI/DRD/Afilias/Shared.pm [hunk 2]
```diff
@@ -141,4 +141,14 @@ sub verify_name_domain
 }
 
 ####################################################################################################
+####################################################################################################
```
Comment (thedarkwinter) [inline lib/Net/DRI/DRD/Afilias/Shared.pm:144]: You seem to have moved PRO into Donuts, but the Pro Extension functions into Afilias::Shared. Can you clarify what's happening here? Is the PRO extension not supported in Donuts? In which case, is this functionality deprecated completely?

**Review by thedarkwinter — CHANGES_REQUESTED** [review]

File: lib/Net/DRI/DRD/Afilias/Shared.pm (hunk 2 above)
Comment (pc-stein) [reply lib/Net/DRI/DRD/Afilias/Shared.pm:144]: According to Paulo Jorge, we moved the Pro to Shared and would cleanup at a later date after discussion. Donuts doesn't support the Pro extension.

File: lib/Net/DRI.pm (hunk 1 above)
Comment (pc-stein) [reply lib/Net/DRI.pm:287]: No, Afilias extensions are not suported in Donuts.Ill remove the comment and put it in the DRD.

**Pull request merged**